e2e/tests/account-settings.spec.ts

```diff
@@ -0,0 +1,122 @@
+      await accountPage.clearAllRequiredFieldsForProfile();
+      await accountPage.profileSaveBtn().click();
+      await expect(page.getByText('This field is required')).toHaveCount(3);
+    });
+
+    test('Verify user can update personal information', async ({ accountPage }) => {
+      const newFirstName = await accountPage.changeProfileFirstName();
+      const newLastName = await accountPage.changeProfileLastName();
+      const newPhoneNumber = await accountPage.changePhoneNumber();
+
+      await accountPage.saveProfileUpdates();
+      await accountPage.verifyInputValue({ locator: accountPage.profileFirstNameInput(), text: newFirstName });
+      await accountPage.verifyInputValue({ locator: accountPage.profileLastNameInput(), text: newLastName });
+      await accountPage.verifyPhoneNumberInputValue(newPhoneNumber);
+    });
+  });
+
+  test.describe('Account / Shipping Address', () => {
+    test.beforeEach('Navigate to the Account settings', async ({ accountPage }) => {
+      await accountPage.navigateToAccountPage();
+    });
+
+    test('Verify red messages appear if required fields are not filled in', async ({ page, accountPage }) => {
+      await accountPage.clearAllRequiredFieldsForShipping();
+      await accountPage.shippingSaveBtn().click();
+      await expect(page.getByText('This field is required')).toHaveCount(5);
+    });
+
+    // BUG: https://app.shortcut.com/takeshape/story/12729/shipping-address-data-are-not-saved-after-refreshing
+    test.skip('Verify user can update shipping address', async ({ page, accountPage }) => {
+      let firstName: string;
+      let lastName: string;
+      let address: string;
+      let city: string;
+      let postalCode: string;
+      let country: string;
+      let state: string;
+
+      await test.step('Update required fields', async () => {
+        firstName = await accountPage.changeShippingFirstName();
+        lastName = await accountPage.changeShippingLastName();
+        address = await accountPage.changeShippingAddress1();
+        city = await accountPage.changeShippingCity();
+        postalCode = await accountPage.changeShippingPostalCode();
+      });
+
+      await test.step('Update selection fields', async () => {
+        country = await accountPage.selectRandomCountry();
+        state = await accountPage.selectRandomState();
+      });
+
+      await test.step('Save and verify changes', async () => {
+        await accountPage.saveShippingUpdates();
+        // BUG: Data are not saved after reloading, see link above
+        await page.reload();
+        await page.waitForLoadState('domcontentloaded');
+        await accountPage.verifyShippingChangesSaved({
+          firstName,
+          lastName,
+          address,
+          city,
+          postalCode,
+          country,
+          state
+        });
+      });
+    });
+  });
+
+  test.describe('Password', () => {
```

Comment:
Should add a success case too

Reply:
I had some doubts about it: we're using only one test user for all authenticated tests (account settings, login, product review), and if we change the password, we should change it back.
I see the number of workers is set to 1, so I can add a test step that will change the password back. But if we increase the number of workers in the future, it will bring some flakiness, because there is a possibility that we'll try to log in with the old password in some test while we're doing the password change.

Reply:
I've added a positive test here and also did the same for the email in the Profile test. Please, review it again.

Reply:
I understand the concern, it seems it would be best to create a new user for just the password test. If you feel that is beyond the scope of this PR I suggest leaving it out for now and making a story for it.

Reply:
Then I'll put a fixme annotation on this test, but I'm not sure if we can register new users on penny because of the reCAPTCHA.